src/Controller/AgendaCaseItemController.php

```diff
@@ -93,7 +93,11 @@ public function caseItemDocuments(Agenda $agenda, AgendaCaseItem $agendaItem): R
     {
         $this->denyAccessUnlessGranted('view', $agendaItem);
 
-        $documents = $agendaItem->getDocuments();
+        $documents = $agendaItem->getDocuments()->toArray();
+
+        uasort($documents, function (Document $a, Document $b) {
+            return $a->getUploadedAt() <=> $b->getUploadedAt();
+        });
```

Comment:
Consider using `ORM\OrderBy` on [AgendaCaseItem::documents](https://github.com/itk-dev/naevnssekretariatet/blob/develop/src/Entity/AgendaCaseItem.php#L30) to do this in the database query (cf. https://symfonycasts.com/screencast/doctrine-relations/orderby-extra-lazy#ordering-question-gt-getanswers-with-orm-orderby).

Reply:
Used in 64c80fb74d06ae7b45b96270d32692d911a39b01!